Dockerfile

```diff
@@ -1,57 +1,5 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
```

Comment:
It is not stable yet, the release is scheduled within one week (April 21st). By then, hopefully this PR will be fully mature and ready to use.

![xerus-release-date](https://cloud.githubusercontent.com/assets/324683/14545390/6e1df568-0268-11e6-9667-4d7803bd7072.jpg)